bin/v-add-csf-firewall

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#Author : vvcares.com
```

Comment:
Can be solved with the https://github.com/hestiacp/hestiacp/pull/3694/commits/c5d58dc5b91d2531b3f7e804916223f66d097518 

That will be added in 1.9 with a custom plugin that will handle everything